modelerfour/modeler/modelerfour.ts

```diff
@@ -595,6 +644,14 @@ export class ModelerFour {
             schema.type = OpenAPI.JsonType.Object;
             break;
           }
+
+          {
+            // no type info at all!? 
+            // const err = `The schema '${name}' has no type or format information whatsoever. ${this.location(schema)}`;
+            this.session.warning(`The schema '${name}' has no type or format information whatsoever. ${this.location(schema)}`, ['Modeler', 'MissingType'], schema);
+            // throw Error(err);
```

Comment:
I suppose this is still here in case you later decide to throw an error instead?

Reply:
Yeah; kinda. I don't like `AnySchema ` for this all that much, but I may be the price to pay.